### PR TITLE
bump version to 0.1.0

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

### CODE REVIEW

File: extension/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "repository": {
         "url": "https://github.com/Microsoft/vscode-java-test"
     },
-    "version": "0.0.1",
+    "version": "0.1.0",
```
Comment (yaohaizh) [inline extension/package.json]: Should the internal java package also need changes?

**Comment by ansyral** [comment]
leave out the change of package.lock.json?

File: extension/package.json (hunk 1 above)
Comment (testforstephen) [reply extension/package.json]: java package has already been changed to 0.1.0.

Comment (ansyral) [reply extension/package.json]: already updated

**Pull request merged**